feature/auth/src/main/java/com/teamoffroad/feature/auth/presentation/AuthScreen.kt

```diff
@@ -51,14 +47,6 @@ internal fun AuthScreen(
     viewModel: AuthViewModel = hiltViewModel(),
 ) {
     val isAuthUiState by viewModel.authUiState.collectAsStateWithLifecycle()
```

Comment:
```suggestion
    val authUiState by viewModel.authUiState.collectAsStateWithLifecycle()
```
uiState 자체의 네이밍은 그대로 사용하는게 좋을 것 같아요!

Reply:
수정했습니다~